versioned_docs/version-8.7/self-managed/identity/authentication.md

```diff
@@ -0,0 +1,52 @@
+---
+id: authentication
+title: "Authentication"
+sidebar_label: "Authentication"
+description: "Learn how Identity authenticates users or clients via log-in screen or M2M tokens."
+---
+
+Based on your [configuration](/self-managed/identity/configuration/identity-configuration-overview.md), Users and Applications interacting with Camunda 8 will authentication via the respective IdP following the [OAuth 2.0 protocol](https://oauth.net/2/), either by using a log-in page or by using [M2M tokens](#m2m-machine-to-machine-authentication).
+
+## Log-in page authentication
+
+Identity will show the log-in page by the respective IdP to your users when they access one of the Camunda web applications.
+
+## M2M (machine-to-machine) authentication
+
+### Prerequisites
+
+- A running [Identity](/self-managed/identity/what-is-identity.md) service
+- An [application](/self-managed/identity/application-user-group-role-management/applications.md) for your service
+- The client ID of your application
+- The client secret of your application
+- A REST client of your choice
+
+### Generate token
+
+In our example, the Keycloak instance that supports Identity can be found via `http://localhost:18080`.
+This may be different for you, so adjust the host name (and port if required) as appropriate.
+
+To request a token, use the following cURL command replacing the placeholders with your applications
+details:
+
+```
+curl --location --request POST 'http://localhost:18080/auth/realms/camunda-platform/protocol/openid-connect/token' \
+--header 'Content-Type: application/x-www-form-urlencoded' \
+--data-urlencode 'client_id=[CLIENT_ID]' \
+--data-urlencode 'client_secret=[CLIENT_SECRET]' \
+--data-urlencode 'grant_type=client_credentials'
+```
+
+A successful authentication response looks like the following:
+
+```
+{
+  "access_token": "<TOKEN>",
+  "expires_in": 300,
+  "refresh_expires_in": 0,
+  "token_type": "Bearer",
+  "not-before-policy": 0
+}
+```
+
+You can use the token to authenticate an application with Camunda components, for example to access REST APIs of Operate.
```

Comment:
I think this might not be sufficient for the average starting out user. They'd need some pointer how to set this as a HTTP header or so. Ideally we'd have some kind of guide that we can link to.

That said, it's fine for me to leave things as is for now if we want to make progress with getting this merged.

Reply:
Leaving as is for now.